lnrpc/rpc.proto

```diff
@@ -1878,6 +1878,9 @@ message NetworkInfo {
     // The number of edges marked as zombies.
     uint64 num_zombie_chans = 11 [json_name = "num_zombie_chans"];
 
+    // Whether we consider ourselves synced with the public channel graph.
+    bool synced_to_graph = 12 [json_name = "synced_to_graph"];
```

Comment:
been thinking about it, I think it’d be best if this were on getinfo. this call is way more expensive, and would severely limit the ability to long poll. no reason to make two rpc calls to figure out if we’re synced

Reply:
Done.